pkgs/by-name/al/alacritty/package.nix

```
@@ -44,7 +44,7 @@ let
 in
 rustPlatform.buildRustPackage (finalAttrs: {
   pname = "alacritty";
-  version = if !withGraphics then "0.16.1" else "0.16.0-graphics";
+  version = if !withGraphics then "0.16.1" else "0.16.1-graphics";
```

Does the update script even work I wonder?

If it doesn't, a solution like this could fix this.


```suggestion
  version = if !withGraphics then finalAttrs.mainVersion else "${finalAttrs.mainVersion}-graphics";
  mainVersion = "0.16.1";
```

---

Thanks for pointing this out, @JohnRTitor, I'll check

---

Happy to report that the update script works. Tested using the following command, which updated the appropriate version for the package:

```
nix-shell maintainers/scripts/update.nix --argstr package alacritty
nix-shell maintainers/scripts/update.nix --argstr package alacritty-graphics
```